entrypoint.sh

```diff
@@ -1,27 +1,18 @@
 #!/bin/sh -l
 
 # Remove Docusaurus default README.md
-rm /app/docs/README.md
+# rm /app/docs/README.md
```

Comment:
Please revert

Reply:
Coming back to this PR, I was confused and am confused because even when #46 removes the README. It's still showing up here? https://docusaurus-builder.pages.dev/  How do we explain that? 

Additionally, why is the demo README file even there? We don't need it, do we? Our docusaurus-builder project website's root README is the one we want to show.

Reply:
https://docusaurus-builder.pages.dev/ also on this page is the mermaid diagram even working?

Reply:
New commits has more docs, and no demo README.md in docs folder

Reply:
@Bucknalla need you to dismiss your review to merge this if you don't have comments.

Reply:
@vipulgupta2048 try to build the handbook repo locally and see what happens with this change.

Reply:
So how can we locally test this repo is working now? There's no README to test if docusaurus-builder is behaving. I'm not against this change to completely remove the README but we should be able to quickly verify that docusaurus-builder is working as expected, without having to test it on another repo.

Reply:
@vipulgupta2048

Reply:
I thought we previously identified this already that this repository can't be used to test new changes fully. Any new changes made to docusaurus-builder will first need to be merged and updated in Flowzone before they can get tested. If the intention is to test this repository as well as develop features, then we can have a separate discussion.

Reply:
Additionally, with the intention to add docs to this repository to help users with docusaurus-builder. I say we focus on documenting this repository first which means for us to not have a custom docs README at this point.